test/multivec_test.py

```diff
@@ -10,27 +11,38 @@
 
 def test_bedfile_to_multivec():
     runner = clt.CliRunner()
-    input_file = op.join(testdir, 'sample_data', 'sample.bed.gz')
-    tempfile.NamedTemporaryFile(delete=False)
-    # TODO: Make assertions about result
-    # print("input_file", input_file)
-
-    result = runner.invoke(
-        ccc.bedfile_to_multivec,
-        [input_file,
-         '--has-header',
-         '--assembly', 'hg38',
-         '--base-resolution', '10'])
-
-    # import traceback
-    a, b, tb = result.exc_info
-    '''
-    print("exc_info:", result.exc_info)
-    print("result:", result)
-    print("result.output", result.output)
-    print("result.error", traceback.print_tb(tb))
-    print("Exception:", a,b)
-    '''
+    input_file = op.join(testdir, 'sample_data', 'sample.multival.bed')
+    chromsizes_file = op.join(testdir, 'sample_data', 'sample.multival.chrom.sizes')
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        out_file = op.join(tmp_dir, 'out.multivec')
+        # TODO: Make assertions about result
+        # print("input_file", input_file)
+        # print("output_file", out_file.name)
+
+        _ = runner.invoke(
+            ccc.bedfile_to_multivec,
+            [input_file,
+             '--output-file', out_file,
+             '--assembly', 'hg38',
+             '--num-rows', 3,
+             '--chromsizes-filename', chromsizes_file,
+             '--starting-resolution', '1000'])
+
+        # import traceback
+        # a, b, tb = result.exc_info
+
+        # print("exc_info:", result.exc_info)
+        # print("result:", result)
+        # print("result.output", result.output)
+        # print("result.error", traceback.print_tb(tb))
+        # print("Exception:", a,b)
+
+        tsinfo = ctv.tileset_info(out_file)
+        # print("tsinfo:", tsinfo)
+
+        assert 'resolutions' in tsinfo
+        assert tsinfo['max_pos'][0] == 18000
```

Comment:
Since the sample files are small, could you just say exactly what you expect to have returned, and if anything is different, you'll catch it?

Reply:
Yup, good suggestion! I added some more checks of the returned tile value but stopped a little short of checking for the exact values.